metrics-core/src/main/java/software/amazon/swage/metrics/record/cloudwatch/MetricDataAggregator.java

```diff
@@ -41,9 +42,8 @@
  * by the recorder, and does not currently support "general purpose"
  * aggregation of metric data/MetricDatums.
  * <p>
- * Metrics are aggregated by namespace, metricName, attributes, and unit.
- * Matching metrics events will be added together to form a single
- * {@link StatisticSet}.
+ * Metrics are aggregated by namespace, metricName, attributes, unit, and value.
+ * Raw metrics are published so percentiles are available in CloudWatch.
```

Comment:
This is not correct. This publishes raw metrics if only one is recorded otherwise you publish a statistics set where the min and max are equivalent. Which is one of the cases that cloudwatch handles differently. Also this class does not publish it simply aggregates.

Reply:
Okay. I'll go with a small change to the original

> Matching metrics events will be added together to form a single *value*.